src/app/stack/utils/report-summary-utils.ts

```diff
@@ -83,80 +83,126 @@ export class ReportSummaryUtils {
 
             let securityIssues: number = 0;
             let dependenciesEffected: number = 0;
-            let maxIssue: SecurityInformationModel = null,
-                temp: SecurityInformationModel = null;
+            // let maxIssue: SecurityInformationModel = null,
+            //     temp: SecurityInformationModel = null;
 
             let analyzedDependencies: Array<ComponentInformationModel>;
+            let publicVulnerabilitiesCount: number = 0;
+            let privateVulnerabilitiesCount: number = 0;
```

Comment:
Declare at the place where you assign the value to it.

Reply:
fixed